core/lib/mcxtx.h

```diff
@@ -256,5 +256,11 @@
      * bytes.
      */
 #    endif
-    dr_zmm_t simd[MCXT_NUM_SIMD_SLOTS];
+    union {
+        dr_zmm_t simd[MCXT_NUM_SIMD_SLOTS];
+#    ifdef AVOID_API_EXPORT
+        /* The ymm field is provided for backward compat. */
```

Comment:
I would make this the exposed doxygen comment for this field and include "an alias for the `simd` field" or sthg so we don't have to duplicate the big comment on simd.

Reply:
Done. I've also added a release note about the name change (# links do not apply to structure fields)